src/tallies/tally.cpp

```diff
@@ -250,6 +254,26 @@ Tally::Tally(pugi::xml_node node)
     }
   }
 
+  // Currently the pulse-height tally is only working in a selected type of
+  // settings Here these settings are checked again
+  bool exists = std::find(std::begin(scores_), std::end(scores_), ReactionType::PHT) !=
+                std::end(scores_);
+  if (exists && scores_.size() > 1) {
+    fatal_error("Error: The Pulse-Height Tally can currently not be used in combination with other tallys.");
+  }
+  if (exists && !settings::photon_transport) {
+      fatal_error("Error: The Pulse-Height Tally works only with photon transport True.");
+  }
+#ifdef _OPENMP
+  int threads = omp_get_max_threads();
+#endif
+  if (exists&& threads != 1) {
+    fatal_error("Error: The Pulse-Height Tally works currently only in a single thread simulation.");
+  }
```

Comment:
Unfortunately this is a show stopper for having a feature accepted in OpenMC. The default build includes OpenMP threading, and virtually every CPU these days has multiple cores.

Reply:
Yes, I checked this already last night, and it works with multiple threads. So it shouldn't be an issue anymore :-)